### ISSUE TITLE
Examples for model training?

### ISSUE THREAD

**Issue by penfever** [quasimetric-learning/torch-quasimetric#1]
Hello!  I am interested in utilizing quasimetric distance measures with contrastive vision-language models. Can you please add some simple examples to your readme for how you recommend incorporating d_iqe(x[:, None], y) or d_iqe(x, y) into existing contrastive loss functions such as InfoNCE loss?

**Comment by ssnl** [comment]
Hi @penfever Thanks for your interests! To my best knowledge there hasn't been researches on how to best do contrastive learning with quasimetrics, so I am hesitant to make any recommendations in README. However, a starting point might be just using the negated quasimetric distance (or some transformations) of it as logits in NCE loss. Hope that this helps!
